Realm/RLMObject.mm

```diff
@@ -40,7 +40,11 @@ - (instancetype)init {
 }
 
 - (instancetype)initWithValue:(id)value {
-    return [super initWithValue:value schema:RLMSchema.sharedSchema];
+    RLMSchema *schema = RLMSchema.partialSharedSchema;
+    if (![schema schemaForClassName:self.class.className]) {
```

Comment:
@jpsim they are cached? The idea is to make it _possible_ to avoid eagerly loading in every binary, because that can take up several megs